client/src/featureform/register.py

```diff
@@ -2572,6 +2572,50 @@ def apply(self):
         else:
             state().create_all(self._stub)
 
+    def wait(self, resource_type, name, variant=None, timeout=None):
```

Comment:
Should this return anything?

Reply:
Why not use self.type, self.name, self.variant etc?

Reply:
it does use self.type, variant, etc in the wrapper for each resource in resources.py